### PR TITLE
Removing command redundant arguments in get.go

### PR DESCRIPTION
**Pull request by pswica**
**What type of PR is this?**
/kind cleanup
**What this PR does / why we need it**:
It removes redundant command arguments in `kubectl/pkg/cmd/get/get.go`
**Which issue(s) this PR fixes**:
**Special notes for your reviewer**:
I was motivated by the following comment in the source code:
`// TODO: remove the need to pass these arguments, like other commands.`
**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/get/get.go [hunk 1]
```diff
@@ -436,12 +441,12 @@ func (o *GetOptions) transformRequests(req *rest.Request) {
 
 // Run performs the get operation.
 // TODO: remove the need to pass these arguments, like other commands.
```
Comment (draveness) [inline pkg/kubectl/cmd/get/get.go]: nit: We could remove the TODO along with this commit.

**Review by draveness — COMMENTED** [review]
/ok-to-test

File: pkg/kubectl/cmd/get/get.go (hunk 1 above)
Comment (pswica) [reply pkg/kubectl/cmd/get/get.go]: Thanks a lot for your always helpful review. I was thinking of doing this, but I came to the conclusion that there is still work to be done: getting rid of the `cmd` argument. This is a a more significant refactor, however, so I opted to leave that for another commit.

File: pkg/kubectl/cmd/get/get.go [hunk 2]
```diff
@@ -163,12 +165,14 @@ func NewCmdGet(parent string, f cmdutil.Factory, streams genericclioptions.IOStr
...
 			cmdutil.CheckErr(o.Validate(cmd))
-			cmdutil.CheckErr(o.Run(f, cmd, args))
+			cmdutil.CheckErr(o.Run(cmd, args))
 		},
 		SuggestFor: []string{"list", "ps"},
 	}
 
 	o.PrintFlags.AddFlags(cmd)
+	rc := f.RESTClient
+	o.RESTClient = rc
```
[the remark this answers is not in the document]
Comment (pswica) [reply pkg/kubectl/cmd/get/get.go]: Great suggestion. I was experimenting with two return values previously which this is a relic of. I will apply your suggested changes and re-push.

[the remark this answers is not in the document]
Comment (pswica) [reply pkg/kubectl/cmd/get/get.go]: Great suggestion. I was experimenting with two return values previously which this is a relic of. I will apply your suggested changes and re-push.

**Pull request closed without merging**